pythonforandroid/bootstraps/common/build/jni/application/src/start.c

```diff
@@ -71,7 +71,7 @@ int file_exists(const char *filename) {
 }
 
 /* int main(int argc, char **argv) { */
-int main(int argc, char *argv[]) {
+int main_(int argc, char *argv[], int call_exit) {
```

Comment:
What happened to the main?
I've never played with that part, but won't this get linking issues that the main is not exposed?

Reply:
main only gets called from within the JNICALL functions, and my editor complained about the signature. so i renamed it, maybe start_python or so woud better fit.